api/src/pcapi/core/offerers/repository.py

```diff
@@ -581,3 +583,17 @@ def get_emails_by_offerer(offerer: models.Offerer) -> set[str]:
     }
     emails.discard(None)
     return emails
+
+
+def find_venues_of_offerer_from_siret(siret: str) -> Tuple[Offerer | None, list[models.Venue]]:
+    siren = siret[:9]
+    offerer = models.Offerer.query.filter(models.Offerer.siren == siren).one_or_none()
+    venues = (
+        models.Venue.query.join(models.Offerer)
+        .filter(models.Offerer.siren == siren, models.Venue.isPermanent.is_(True))
+        .order_by(models.Venue.publicName)
+        .all()
+        if offerer
+        else []
+    )
```

Comment:
(style / question de goût) Sortir "rapidement" de la fonction me semble plus lisible que cette condition sur plusieurs lignes (et on voit clairement la valeur de sortie quand on ne trouve pas la structure) : 

```suggestion
    if not offerer:
        return None, []
    venues = (
        models.Venue.query.join(models.Offerer)
        .filter(models.Offerer.siren == siren, models.Venue.isPermanent.is_(True))
        .order_by(models.Venue.publicName)
        .all()
    )
```